integration/kubernetes/k8s-volume.bats

```diff
@@ -17,11 +17,12 @@ setup() {
 	get_pod_config_dir
 
 	tmp_file=$(mktemp -d /tmp/data.XXXX)
+	pod_yaml=$(mktemp --tmpdir pod_config.XXXXXX.yaml)
 	msg="Hello from Kubernetes"
 	echo $msg > $tmp_file/index.html
 	pod_name="pv-pod"
 	# Define temporary file at yaml
-	sed -i "s|tmp_data|${tmp_file}|g" ${pod_config_dir}/pv-volume.yaml
+	sed -e "s|tmp_data|${tmp_file}|g" ${pod_config_dir}/pv-volume.yaml > "$pod_yaml"
```

Comment:
do we remove that tmpfile later on? I don't see it in the patch. maybe we need a bash trap exit hook to do that?

Reply:
good catch, fixed. The `teardown` will ensure to delete the file even if something fails in the test.